packages/client/components/kit/SvgImage.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { View } from 'react-native';
+import { SvgUri } from 'react-native-svg';
+import { SvgImageProps } from './SvgImage.contract';
+
+const SvgImage: React.FC<SvgImageProps> = ({
+    uri,
+    accessibilityLabel,
+    style,
+}) => {
+    return (
+        <View style={style}>
+            <SvgUri uri={uri} accessibilityLabel={accessibilityLabel} />
```

Comment:
`SvgUri` component did not work from the first time on the Web, so I decided do use a usual `Image` component on the Web.

Reply:
**👏 praise:** I think this a great idea